core/src/com/unciv/models/stats/Stat.kt

```diff
@@ -2,13 +2,22 @@ package com.unciv.models.stats
 
 import com.unciv.logic.civilization.NotificationIcon
 import com.unciv.models.UncivSound
+import com.unciv.ui.utils.Fonts
 
-enum class Stat(val notificationIcon: String, val purchaseSound: UncivSound) {
-    Production(NotificationIcon.Production, UncivSound.Click),
-    Food(NotificationIcon.Food, UncivSound.Click),
-    Gold(NotificationIcon.Gold, UncivSound.Coin),
-    Science(NotificationIcon.Science, UncivSound.Chimes),
-    Culture(NotificationIcon.Culture, UncivSound.Paper),
-    Happiness(NotificationIcon.Happiness, UncivSound.Click),
-    Faith(NotificationIcon.Faith, UncivSound.Choir);
-}
\ No newline at end of file
+enum class Stat(
+    val notificationIcon: String,
+    val purchaseSound: UncivSound,
+    val character: Char,
+    val isPotentialCurrency: Boolean = true
```

Comment:
Character, maybe. But this is definitely not a 'basic value' of the stat, this is totally gameplay-related. This should not be a field.
The question then becomes where to put it, which becomes - where is this actually used?

Reply:
Oh, totally forgot about this. I had another approach coded first, quite straightforward - originally a listOf() was instantiated every use, once per Unciv run is enough so just move that to a singleton - Stat.Companion would be a good place IMHO. A companion closer to the use - city screen package only so far - would still be a source screen away at least, so I'd tend to Stat.kt...